lib/gollum-lib/redirects.rb

```diff
@@ -33,6 +33,10 @@ def dump
       @wiki.overwrite_file(REDIRECTS_FILE, self.to_yaml, {})
     end
     
+    def get_head_sha
+      @wiki.repo.head.nil? ? nil : @wiki.repo.head.commit.sha
```

Comment:
Slightly more efficient by not checking explicitly for nil:
`@wiki.repo.head ? @wiki.repo.head.commit.sha : nil`

Reply:
Thanks!